src/kOS/Function/Suffixed.cs

```diff
@@ -197,6 +197,17 @@ public override void Execute(SharedObjects shared)
         }
     }
 
+    [Function("lex", "lexicon")]
+    public class FunctionLexicon : FunctionBase
+    {
+        public override void Execute(SharedObjects shared)
+        {
+            AssertArgBottomAndConsume(shared);
+            var listValue = new Lexicon<object,object>();
+            ReturnValue = listValue;
+        }
+    }
+
```

Comment:
I never did like calling it "lexicon", but it's too late to change it now.  (It implies a text-based lookup (lexicographical), which implies the keys are strings, which they don't have to be).